test/common/http/utility_test.cc

```diff
@@ -653,6 +653,32 @@ TEST(HttpUtility, parseHttp2Settings) {
               http2_options.max_inbound_window_update_frames_per_data_frame_sent().value());
   }
 
+  {
+
+    TestScopedRuntime scoped_runtime;
+    scoped_runtime.mergeValues({{"envoy.reloadable_features.safe_http2_options", "false"}});
```

Comment:
Why this is testing the false case?

Reply:
I have updated the value of the `DEFAULT_MAX_CONCURRENT_STREAMS` value directly and keep the previous value in `DEFAULT_MAX_CONCURRENT_STREAMS_LEGACY`, I only need to test the the legacy value is used when the runtime flag is set to false.

The original test could cover the new default value.